package.json

```diff
@@ -136,5 +138,8 @@
   },
   "prisma": {
     "seed": "tsx prisma/seed.ts"
+  },
+  "volta": {
+    "node": "18.16.0"
```

Comment:
I don't think I'm ready to commit to volta just yet. Let's remove this.

Reply:
@kentcdodds Oh yea this is my bad, I'll make sure to remove it.

Reply:
Reminder ☝️  :) Thanks!